src/askui/android_agent.py

```diff
@@ -60,21 +60,22 @@ def __init__(
         configure_logging(level=log_level)
         self.os = PpadbAgentOs()
         self._reporter = CompositeReporter(reporters=reporters)
-        android_os_handler = AndroidAgentOSHandler(self.os, self._reporter)
+        self._act_agent_os_handler = AndroidAgentOsFacade(self.os, self._reporter)
+        self.act_tool_collection = ToolCollection(
```

Comment:
```suggestion
        self.tools = ToolCollection(
```

Why the act prefix? Is there another tool collection or other tools that we need to differentiate from?